src/cljs/rems/config.cljs

```diff
@@ -70,3 +70,18 @@
          {:params {:disabled true :archived true}
           :handler #(rf/dispatch-sync [:loaded-organizations %])
           :error-handler (flash-message/default-error-handler :top "Fetch organizations")}))
+
+(rf/reg-sub
+ :handled-organizations
+ (fn [db _]
+   (:handled-organizations db)))
+
+(rf/reg-event-db
+ :loaded-handled-organizations
+ (fn [db [_ organizations]]
+   (assoc db :handled-organizations organizations)))
+
+(defn fetch-handled-organizations! []
+  (fetch "/api/organizations/handled"
+         {:handler #(rf/dispatch-sync [:loaded-handled-organizations %])
+          :error-handler (flash-message/default-error-handler :top "Fetch handled organizations")}))
```

Comment:
looks like this is now unused function, due to setting handled orgs in html?

Reply:
Yeah, I basically left this one because I wasn't sure should it be called in some cases to reload the initial data. Now the handled organizations won't reload unless you reload the page, but if we had a  return channel (generalized from multiplayer branch) then it would be simple to call this at the appropriate time. The code itself doesn't really have muvh value but at least it is consistent with the rest of the application although currently unused.